nncf/config/schema.py

```diff
@@ -504,6 +506,7 @@ def with_attributes(schema: Dict, **kwargs) -> Dict:
 }
 
 QUANTIZATION_ALGO_NAME_IN_CONFIG = "quantization"
+EXPERIMENTAL_QUANTIZATION_ALGO_NAME_IN_CONFIG = "experimental_quantization"
```

Comment:
I would prefer to keep experimental config options in a separate file. Is it possible to set up an experimental-only portion of schema and then splice it into the production schema at the stage where the NNCFConfig is actually validated?

Reply:
Thanks for the comment! I fixed it.